internal/integration/integration_test.go

```diff
@@ -99,6 +99,36 @@ func TestInterfaceNoFragments(t *testing.T) {
 	assert.Nil(t, resp.Being)
 }
 
+func TestInterfaceListField(t *testing.T) {
```

Comment:
Could you also add an integration test that uses `@genqlient(pointer: true)`? I think such a test is the only place the generated code would be exercises (and where we'd suss out any issues). Is that right?

Reply:
My theory was that any issues there would show up at build-time.  But it's probably better not to be lazy :-) .

Reply:
LOL, you were write, the code *is* actually broken!  Thanks for the prod :-) .